_data/thedrowsywinger/thedrowsywinger.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Abdullah Md. Sarwar # No longer than 18 characters
+institution: North South University 🚩 # no longer than 58 characters
+profile_pic: thedrowsywinger.jpg # Name and extension of your profile picture(ex. mona.png)
+quote: " Quit. Don't quit. Noodles. Don't noodles. You are too concerned with what was and what will be. " # no longer than 100 characters
```

Comment:
Remove double quotes from begin and end.

Reply:
Got it!